bin/opam-web.sh

```diff
@@ -45,27 +32,18 @@ redirect: [
   "https://${BASEURL}/1.2.2" { opam-version < "2.0~" }
 ]
 EOF
-mkdir -p /persist/repo-cache/archives
-mkdir -p /persist/repo-cache/cache
-mkdir -p archives
-opam admin cache -n --link=archives /persist/repo-cache/cache
-rsync -a /persist/repo-cache/cache .
-echo 'archive-mirrors: "cache"' >> repo
-if [ -d /persist/legacy ]; then rsync -a /persist/legacy .; fi
+opam admin cache --link=archives ./cache
 opam admin index --minimal-urls-txt
 
 cp -r /usr/local/share/opam2web/content /tmp/
-git clone -s /persist/blog /tmp/content/blog
+git clone https://github.com/ocaml/platform-blog --single-branch --branch master /tmp/content/blog
```

Comment:
We probably need to watch the platform blog as well in the deployer?

Reply:
Good idea, I'll add that in.